template-theory/index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/grib.css">
+    <link rel="stylesheet" href="./css/form-control.css">
+    <link rel="stylesheet" href="./css/lagre-screen.css">
+    <link rel="stylesheet" href="./css/medium-screen.css">
+    <link rel="stylesheet" href="./css/small-screen.css">
+    <title>Theory - Template </title>
+</head>
+
+<body>
+    <div class="header">
+        <div class="menu">
+            <a href="/" class="logo">Theory</a>
+            <ul class="nav">
+                <li>
+                    <a href="#">Home</a>
+                </li>
+                <li>
+                    <a href="#">Generic</a>
+                </li>
+                <li>
+                    <a href="#">Elements</a>
+                </li>
+            </ul>
+            <a href="#" onclick="openMenuMobile(true)" class="navPanelToggle"><span class="fa fa-bars"></span></a>
+            <div id="nav-mobile">
+                <ul>
+                    <a href="">
+                        <li>Home</li>
+                    </a>
+                    <a href="">
+                        <li>Generic</li>
+                    </a>
+                    <a href="">
+                        <li>Elements</li>
+                    </a>
+                </ul>
+                <span class="close-menu" onclick="openMenuMobile(false)">
+                    <i class="fa fa-times"></i>
+                </span>
+            </div>
+        </div>
+    </div>
+    <div class="banner">
+        <h1>Welcome to Theory</h1>
+        <p>A free responsive HTML5 website template by TEMPLATED</p>
+    </div>
+    <!-- Content One -->
+    <div class="content">
+        <div class="row">
+            <div class="col-lg-4 col-md-12 col-sm-12">
+                <div class="actitcle">
+                    <p class="title-actitcle">Magna tempus sed amet</br> aliquam veroeros</p>
+                    <p class="actitcle-discripton">Morbi interdum mollis sapien. Sed ac risus. Phasellus lacinia, magna
+                        a ullamcorper laoreet,
+                        lectus arcu.</p>
+                    <div class="actitcle-footer">
+                        <a href="#" class="btn bg-blue">More</a>
+                    </div>
+                </div>
+            </div>
+            <div class="col-lg-4 col-md-12 col-sm-12">
```

Comment:
```suggestion
            </div>

            <div class="col-lg-4 col-md-12 col-sm-12">
```
Tách các khối ra cho dễ nhìn